PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/service/AzureSdkCategoryService.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.azuresdk.service;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import com.microsoft.azure.toolkit.intellij.azuresdk.model.AzureSdkCategoryEntity;
+import com.microsoft.azure.toolkit.lib.common.cache.Cacheable;
+import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitRuntimeException;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class AzureSdkCategoryService {
+    private static final ObjectMapper CSV_MAPPER = new CsvMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    private static final String SERVICE_CATEGORY_CSV = "/service-category.csv";
+
+    @Cacheable(value = "azure-sdk-category-entities")
+    @AzureOperation(name = "sdk.load_meta_data", type = AzureOperation.Type.TASK)
+    public static Map<String, List<AzureSdkCategoryEntity>> loadAzureSDKCategories() {
+        try (final InputStream stream = WorkspaceTaggingService.class.getResourceAsStream(SERVICE_CATEGORY_CSV)) {
```

Comment:
should not be `WorkspaceTaggingService`

Reply:
updated in https://github.com/microsoft/azure-tools-for-java/pull/5400